src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -27,9 +27,25 @@
 from turicreate.toolkits._internal_utils import (_raise_error_if_not_sframe,
                                                  _numeric_param_check_range)
 
-def create(dataset, target, feature = None, model = 'resnet-50',
-           validation_set='auto', max_iterations = 10, verbose = True,
-           seed = None, batch_size=64):
+_DEFAULT_SOLVER_OPTIONS = {
+'convergence_threshold': 1e-2,
+'step_size': 1.0,
+'lbfgs_memory_level': 11,
+'max_iterations': 10}
+
+  def create(dataset, target, features=None, model = 'resnet-50',
```

Comment:
There is indentation here. This is a syntax error in Python. The 'def' should be at the very start of the line.